src/Types/Output/InterfaceType.hack

```diff
@@ -4,9 +4,16 @@ use namespace HH\Lib\{Dict, Vec};
 use namespace Slack\GraphQL;
 
 abstract class InterfaceType extends CompositeType {
+    abstract const keyset<string> POSSIBLE_TYPES;
 
     <<__Override>>
     final public function getKind(): GraphQL\Introspection\__TypeKind {
         return GraphQL\Introspection\__TypeKind::INTERFACE;
     }
+
+    <<__Override>>
+    final public function getPossibleTypes(): vec<GraphQL\Introspection\__Type> {
+        return Vec\map(static::POSSIBLE_TYPES, $type ==> $this->schema->getIntrospectionType($type))
```

Comment:
Would it be better to cast the return value of `getIntrospectionType` to `as nonnull`?

Reply:
Yea, I was trying to be safe but it is probably better to fail hard rather than silently failing here.